packages/create-neutrino-project/commands/init/templates/react-router/src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
+import './App.css';
+
+import AsyncFoo from './components/Foo';
```

Comment:
I want to avoid a lot of boilerplate that is going to end up being deleted. Let's give them a single home route that is dynamically imported.